### PR TITLE
Adding holidays for syria

### PR DESCRIPTION
**Pull request by abdulrhmanak213**
I have added the official holidays list specific to Syria

### CODE REVIEW

File: tests/.pest/snapshots/Countries/SyriaTest/it_can_translate_syria_holidays_into_arabic.snap [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        "date": "2024-03-21"
+    },
+    {
+        "name": "Teacher's Day",
+        "date": "2024-03-21"
+    },
+    {
+        "name": "\u0639\u064a\u062f \u0627\u0644\u0641\u0635\u062d \u0627\u0644\u063a\u0631\u0628\u064a",
+        "date": "2024-03-31"
+    },
```
Comment (Nielsvanpach) [inline tests/.pest/snapshots/Countries/SyriaTest/it_can_translate_syria_holidays_into_arabic.snap:17]: A combination of languages here

**Review by Nielsvanpach — CHANGES_REQUESTED** [review]

File: tests/.pest/snapshots/Countries/SyriaTest/it_can_translate_syria_holidays_into_arabic.snap (hunk 1 above)
Comment (abdulrhmanak213) [reply tests/.pest/snapshots/Countries/SyriaTest/it_can_translate_syria_holidays_into_arabic.snap]: I have fix this issue the problem was in the ('s)

File: src/Countries/Syria.php [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+            "Eid al-Fitr" => '04-10',
+            "Syrian Independence Day" => '04-17',
+            "Labor Day" => '05-01',
+            "Eastern Easter" => '05-05',
+            "Martyr\n's Day" => '05-06',
+            "Eid al-Adha" => '06-16',
+            "Islamic New Year" => '07-07',
+            "The commemoration of the birth of the Prophet Muhammad" => '09-15',
+            "The October Liberation War" => '10-06',
+            "Merry Christmas" => '12-25',
```
Comment (Nielsvanpach) [inline src/Countries/Syria.php:30]:
```suggestion
            "Christmas" => '12-25',
```

**Comment by Nielsvanpach** [comment]
Thanks!

**Pull request merged**